config/database.yml

```diff
@@ -8,6 +8,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
+  username: playthisforus
```

Comment:
Sounds good, that's what I've done in my projects in the past, just wasn't sure how to go about it here.

I'll revise the PR when I get back to my computer.